### PR TITLE
Ruben landingpages

### LINKED ISSUE #516: New employers landing page
As an employer
So that I can be more easily persuaded to hire grads
I want a new landing page that explains to me the benefits of hiring from Makersx

On branch ruben-landingpages

Couple of blockers:

1. I'd like that instead of 'City' on the page, its picks up the location based on the browser
2. I believe there's a slight misalignment of a CTA on mobile, unlinked to the work I did
3. Blocked by #511

### PR DESCRIPTION
**Pull request by rubenkostucki**
We just need to change the /employers and /employers1 so we can A/B test the two landingpages.

Closes #516

### CODE REVIEW

**Comment by rubenkostucki** [comment]
closes #516

File: source/employers-1.html.haml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- title "Employers"
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/students-in-lecture.jpg")});"}
+  .container
+    .centered-row
+      %h4.leader Hire London's
+      %h1 top junior developers
+      %p 30 new candidates every 6 weeks
+    .button-row
+      %p
+        = link_to "Hire Developers", "https://makersacademy.typeform.com/to/sz8vTw", class: "employers-button button button--vertical", target: "_blank"
```
Comment (pitchinvasion) [inline source/employers-1.html.haml]: Need to update to new page

File: source/employers-1.html.haml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+%section.backgrounded
+  .container
+    .image-grid-row
+      - fetch_logos.shuffle.each do | logo |
+        = image_tag logo, { alt: alt_text_from_filename(logo), title: alt_text_from_filename(logo) }
+
+= partial :"apply", locals: { page: :employers }
+:javascript
+  analytics.track('Viewed Employers Page');
+  analytics.trackLink(document.getElementsByClassName('employers-button'), 'Viewed Employers Enquiry Form');
```
Comment (pitchinvasion) [inline source/employers-1.html.haml]: This needs to be deleted and use the new data attribute tag instead

**Comment by pitchinvasion** [comment]
Blocked by #514 and by this ticket in applications https://waffle.io/makersacademy/engineering/cards/56b1d7aca5f1a72d0012617c

**Comment by rubenkostucki** [comment]
:shipit:

**Pull request merged**